DataRepo/utils/text_utils.py

```diff
@@ -48,3 +48,11 @@ def autowrap(text: str, default_width: int = 80, **kwargs):
             wrapped += f"{wrappedline}\n"
 
     return wrapped[:-1], nlines, max_width
+
+
+def is_number(val):
+    try:
+        float(str(val))
+    except Exception:
```

Comment:
Suggest using a more precise check, but probably not a big deal.
```suggestion
    except ValueError:
```

Reply:
I'm not sure I can envision how this adds value.  Either it's a number or it's not. I could try and figure out what possible exceptions could get raised here and add them all, but I cannot think of an exception that shouldn't result in anything other than returning False.